tpm/tpm.go

```diff
@@ -74,6 +75,15 @@ func WithDisableDownload() NewTPMOption {
 	}
 }
 
+// WithCapabilities explicits sets the capabilities rather
+// than acquiring them from the TPM directly.
+func WithCapabilities(caps *Capabilities) NewTPMOption {
```

Comment:
Can it be made available in tests only by adding it to `tpm_test.go`? I'm not sure if we want to support it as an actual option. And maybe the property in the options can be named `testCapabilities`, or something like that, to make it explicit.